src/video/camerasource.cpp

```diff
@@ -156,6 +164,8 @@ bool CameraSource::subscribe()
     if (!isOpen)
     {
         ++subscriptions;
+        //qDebug() << "is not open, subscriptions: " << subscriptions;
```

Comment:
Why not now? IMO most of those qDebug's should only be temporarily and locally added to analyse some misbehaviour.